src/crypto/internal/boring/openssl_port_ctr128.c

```diff
@@ -0,0 +1,8 @@
+#include "goboringcrypto.h"
+
+void
+_goboringcrypto_EVP_AES_ctr128_enc(EVP_CIPHER_CTX *ctx, const uint8_t* in, uint8_t* out, size_t in_len)
```

Comment:
should probably be called just _goboringcrypto_EVP_AES_ctr_enc given you can use the same fn for 128,192,256 bit lenght.

Reply:
Agreed, will change.